tests/ecmp/test_fgnhg.py

```diff
@@ -68,6 +73,17 @@ def configure_interfaces(cfg_facts, duthost, ptfhost, ptfadapter, vlan_ip):
     return port_list, ip_to_port, bank_0_port, bank_1_port
 
 
+def generate_fgnhg_config_from_minigraph(duthost):
+    mg_facts = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
+    fgnhg_data = {}
+    fgnhg_data['FG_NHG'] = mg_facts['FG_NHG']
+    fgnhg_data['FG_NHG_MEMBER'] = mg_facts['FG_NHG_MEMBER']
+    fgnhg_data['FG_NHG_PREFIX'] = mg_facts['FG_NHG_PREFIX']
+    logger.info("fgnhg entries programmed to DUT " + str(fgnhg_data))
+    duthost.copy(content=json.dumps(fgnhg_data, indent=2), dest="/tmp/fgnhg.json")
```

Comment:
Why do we need to copy to duthost, here? Isn't FG_NHG already programmed by the minigraph?

Reply:
Removed this function altogether